src/assets/styles/_utilities.scss

```diff
@@ -0,0 +1,43 @@
+@import "variables";
+
+.cursor--pointer {
+  cursor: pointer;
+}
+.card--shadow {
+  margin: 28px 10px 20px;
+  border-radius: 4px;
+  background-color:$white-color;
+  box-shadow: 0 16px 38px -12px rgba(0, 0, 0, .56), 0 4px 25px 0 rgba(0, 0, 0, .12), 0 8px 10px -5px rgba(0, 0, 0, .2);
+}
+.bg--color{
+  color:$white-color;
+}
+.card--color{
+  color: $gray;
+}
+.text--color{
+  color: $light-grey;
+}
+.title--color{
+  color: $yellow;
+}
+.bgTitle-color{
+  background-color: $yellow;
+}
+.header--color{
+  background-color: $purple-color;
+}
+.creative-color{
```

Comment:
give bg if it's for backgroun-color and if it's color only give color--<color-name>